includes/class-wpcp-notices.php

```diff
@@ -273,7 +273,7 @@ public static function spinner_notice() {
 	 * @since 1.3.2
 	*/
    public static function article_notice() {
-	   $notice = __( 'Article search options will be changed in the next version of WP Content Pilot. Bing search will be replaced with Google Custom Search.', 'wp-content-pilot' );
+	   $notice = sprintf( __( 'Article bing search has been replaced with custom Google Search. Please create %scustom search api key%s and %ssearch engine id%s to run article campaigns.', 'wp-content-pilot' ), '<a href="https://pluginever.com/docs/wp-content-pilot/how-to-create-google-custom-search-api/" target="_blank">', '</a>', '<a href="https://pluginever.com/docs/wp-content-pilot/how-to-create-google-search-engine-id/" target="_blank">', '</a>' );
```

Comment:
Bing has been replaced with Custom Google Search. Please create %scustom search api key%s and %ssearch engine id%s to run article campaigns.